engine/src/main/java/graphql/nadel/engine/execution/HydrationInputResolver.java

```diff
@@ -243,7 +243,10 @@ private boolean isBatchHydrationField(HydrationInputNode hydrationInputNode) {
 
         Field originalField = hydrationTransformation.getOriginalField();
         UnderlyingServiceHydration underlyingServiceHydration = hydrationTransformation.getUnderlyingServiceHydration();
-        ServiceExecutionHydrationDetails hydrationDetails = new ServiceExecutionHydrationDetails(underlyingServiceHydration.getTimeout(),underlyingServiceHydration.getBatchSize());
+        ServiceExecutionHydrationDetails hydrationDetails = new ServiceExecutionHydrationDetails(underlyingServiceHydration.getTimeout(),
+                underlyingServiceHydration.getBatchSize(),
+                null,
+                null);
```

Comment:
Since this is dying I dont want to put this into old engine.  We will handle null on the AGG side (eg no metric tags)